driver/csiplugin/controllerserver.go

```diff
@@ -591,20 +592,91 @@ func (cs *ScaleControllerServer) CreateVolume(ctx context.Context, req *csi.Crea
 		scaleVol.LocalFS = scaleVol.VolBackendFs
 	}
 
+	var RCid string // Remote Cluster ID
+	if volFsInfo.Type == filesystemTypeRemote {
+		glog.V(5).Infof("filesystem %s is remotely mounted, getting cluster ID information of the owning cluster.", volFsInfo.Name)
+		clusterName := strings.Split(volFsInfo.Mount.RemoteDeviceName, ":")[0]
+		scaleconfig := settings.LoadScaleConfigSettings()
+
+		for i := range scaleconfig.Clusters {
```

Comment:
Should we break the for loop once we find the required remote cluster id, instead of looping over all the clusters?